UM/Mesh/MeshBuilder.py

```diff
@@ -759,3 +767,31 @@ def addConvexPolygonExtrusion(self, xy_points, y0, y1, color=None):
         self.addQuad(v0, v1, v2, v3, color=color, normal = normal)
 
         return True
+    
+    def _check_file_changed(self, window_state):
```

Comment:
Codestyle; needs to be _checkFileChanged

Reply:
Done in https://github.com/Ultimaker/Uranium/pull/307/commits/269fb932a9a5c667c19400aae202fb40f23b48b4